src/redux/reducers/experiments/experimentsSaved.js

```diff
@@ -1,9 +0,0 @@
-const experimentSaved = (state) => ({
-  ...state,
-  meta: {
-    ...state.meta,
-    saving: false,
-  },
-});
-
```

Comment:
you sure that we can remove this? And there is no need to change the redux store after and experiment is updated? Or are the changes of the store happening from other reducers?

Reply:
nevermind, I see that you moved this to the `experimentsUpdate` reducer